test/methods.jl

```diff
@@ -574,55 +579,55 @@ end
         cropped1 = crop(crop(ga; to=dims(trimmed, X)); to=dims(trimmed, Y))
         _, cropped2 = crop(trimmed, ga)
         cropped_r = crop(ga_r; to=trimmed_r)
-        @test all(cropped .=== cropped1 .=== trimmed)
-        @test all(cropped_r .=== trimmed_r)
+        @test map(identicalelements, maybe_layers.((cropped, cropped1, trimmed))...) |> all
+        @test map(identicalelements, maybe_layers.((cropped_r, trimmed_r))...) |> all
         extended = extend(cropped, ga)[1]
         extended_r = extend(cropped_r; to=ga_r)
         extended1 = extend(extend(cropped; to=dims(ga, X)); to=dims(ga, Y))
         extended_d = extend(cropped; to=ga, filename="extended.tif")
-        @test all(extended .=== extended1 .=== replace_missing(extended_d) .=== ga) 
-        @test all(extended_r .=== ga_r)
+        @test map(identicalelements, maybe_layers.((extended, extended1, replace_missing(extended_d), ga))...) |> all
+        @test map(identicalelements, maybe_layers.((extended_r, ga_r))...) |> all
         @test all(map(==, lookup(extended_d), lookup(extended)))
+    end
 
-        @testset "unformatted dimension works in crop" begin
-            xdim, ydim = X(1.0:0.2:2.0), Y(1.0:1:2.0)
-            A = Raster(rand((X(1.0:0.2:4.0), ydim)))
-            @test lookup(crop(A; to=xdim), X) == 1.0:0.2:2.0
+    @testset "unformatted dimension works in crop" begin
```

Comment:
Indenting changes mean I can't see what you changed

Reply:
It's because all of these testsets were inside a for loop unnecessarily, so I took them out of the for loop.

Reply:
I didn't change anything about the code other than taking it out of the the loop